README.md

```diff
@@ -2017,7 +2022,7 @@ curl -X GET PUT URL -H 'x-api-key: key-abc-123'
 
 ---
 
-#### Remover politica de uso do colaborador por identificador externo
+#### Remover politica de uso dos colaboradores por identificador externo
 
```

Comment:
It would not be better to repeat the observation that the action will affect all users with the same *externalId*?